src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementProductTable.tsx

```diff
@@ -96,7 +96,7 @@ const ReimbursementProductTable: React.FC<ReimbursementProductTableProps> = ({
                   <Box sx={{ display: 'flex', flexWrap: 'wrap', listStyle: 'none', p: 0.5, m: 0 }} component={'ul'}>
                     {uniqueWbsElementsWithProducts.get(key)?.map((product, index) => (
                       <ListItem key={product.index}>
-                        <Box sx={{ display: 'flex', flexDirection: 'row' }}>
+                        <Box sx={{ display: 'flex-row', flexDirection: { sm: 'row', xs: 'column' } }}>
```

Comment:
We want to keep `display` as `flex` since we already specified `flexDirection` as `row` in `sm` and `column` in `xs` screen. Keeping `display` as `flex-row` causes cost and description boxes to align vertically in `sm` screen as the following:

<img width="960" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/91857946/0afb93ae-d7f0-4ee7-9917-10317884329b">